Sources/PubNubChat/PubNub/Models/ChatMessage.swift

```diff
@@ -56,8 +56,8 @@ public struct ChatMessage<Custom: ChatCustomData>: Identifiable, Hashable, Codab
       id: String = UUID().uuidString,
       text: String = String(),
       contentType: String? = nil,
-      content: JSONCodable? = nil,
-      custom: Custom.Message = Custom.Message(),
+      content: Custom.Message.Content? = nil,
+      custom: Custom.Message.Custom =  Custom.Message.Custom(),
```

Comment:
`Custom.Message.Custom` and others like that looks very weird. It could definitely be changed